templates/blog/index.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+  <h1>Blog Posts</h1>
+
+  {% for post in post_list %}
+      <div>
+          <h2><a href="{% url 'blog:post-detail' post.id %}">{{ post.title }}</a></h2>
+          <p>{{ post.content }}</p>
+          <p><strong>Author:</strong> {{ post.author }}</p>
+          <p><strong>Created:</strong> {{ post.created_time }}</p>
+          <p><strong>Comments:</strong> {{ post.comments.count }}</p>
```

Comment:
Verify that 'comments.count' is the correct way to access the number of comments for a post. If the comments are accessed differently, adjust this line accordingly.